### PR TITLE
EL docs update

### PR DESCRIPTION
**Pull request by svlandeg**
## Description
- `DocBin` format
- update of EL docs
- architectures of EL & textcat
- various small fixes

### Types of change
docs

## Checklist
- [x] I have submitted the spaCy Contributor Agreement.
- [ ] I ran the tests, and all new and existing tests passed.
- [ ] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/docs/api/entitylinker.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ api_string_name: entity_linker
 api_trainable: true
 ---
 
+An Entity Linker component disambiguates textual mentions (tagged as named
```
Comment (ines) [inline website/docs/api/entitylinker.md:12]: We should probably decide on either "The entity linker" or "The `EntityLinker`"

File: website/docs/api/entitylinker.md [hunk 2]
```diff
@@ -9,6 +9,12 @@ api_string_name: entity_linker
...
+entities) to unique identifiers, grounding the named entities into the "real
+world". It requires a Knowledge base, a function to generate plausible
```
Comment (ines) [inline website/docs/api/entitylinker.md:14]: I think same here – either knowledge base or `KnowledgeBase`

File: website/docs/api/entitylinker.md (hunk 1 above)
Comment (svlandeg) [reply website/docs/api/entitylinker.md]: You mean that those two variants are OK, but not the mixture "Entity Linker", right? (I agree, just want to double check you don't want to reduce it down to only one of the two options across the docs)

Comment (ines) [reply website/docs/api/entitylinker.md]: Yeah, exactly, sorry if I phrased this in a confusing way!

Comment (svlandeg) [reply website/docs/api/entitylinker.md]: Ok, I think the edit I made is fine then?

**Pull request merged**